tool/state-processor/src/tests.rs

```diff
@@ -318,8 +318,10 @@ fn ring_total_issuance() {
 			&mut migrated_total_issuance,
 		);
 
-		// FIXME: https://github.com/darwinia-network/darwinia-2.0/issues/244
-		// assert_eq!(migrated_total_issuance, solo_issuance * GWEI + para_issuance);
+		assert_eq!(
+			migrated_total_issuance - 155_223_151_710_u128,
```

Comment:
I've considered this solution before. Because the on-chain crab remaining balance still varies now, this may be broken after the exported crab state is updated.

Reply:
I don't care about the updating.

We need a 100% matching here.

Reply:
Okay. This is easy to fix later.